examples/environment-guide/backend-rc.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v1beta3
+kind: ReplicationController
+metadata:
+  labels:
+    type: backend-type
```

Comment:
So, a problem I've had with all the examples, is that they use the same name for everything. The pod name, service name, labels called "name:" for all those, etc. Then you could get confused with selectors, are the matching the name? label? I thought it would be good to give different identifiers so you can see the different "names" and labels.